### ISSUE TITLE
Feature request: Stacking wizardry syringes

### ISSUE THREAD

**Issue by ZephyrWindSpirit** [thephoenixlodge/Heavens-of-Sorcery#136]
When we're first in early game, the reward syringes stack. But both full and empty syringes don't stack normally. We would like to see this be a thing. Please and thank you.

**Comment by Goretantath** [comment]
> 
> 
> When we're first in early game, the reward syringes stack. But both full and empty syringes don't stack normally. We would like to see this be a thing. Please and thank you.

I agree for at-least the empty syringes(the idea of carrying a stack of empty syringes and a stack of mana orbs seems balanced). But im all for making both stack if thats not too op.
